gradle/verification-metadata.xml

```diff
@@ -2571,7 +2574,9 @@
       </component>
       <component group="org.jfrog.buildinfo" name="build-info-extractor-gradle" version="5.2.5">
          <artifact name="build-info-extractor-gradle-5.2.5.jar">
-            <sha256 value="11dfff60a6b351034a589e08f148c1007ec24a08237b62e5c0c8be043b723f64" origin="Verified"/>
+            <sha256 value="11dfff60a6b351034a589e08f148c1007ec24a08237b62e5c0c8be043b723f64" origin="Verified">
+               <also-trust value="a06c1956fec972926f11b1f2ead50bf42366f3e36a72e86efedec887e54049d1"/>
```

Comment:
Why are we keeping this "also-trust" value when the sha256 above is already valid here?